### PR TITLE
Update messages.po

### PR DESCRIPTION
**Pull request by Sh0ham**

### CODE REVIEW

File: src/cryptoadvance/specter/translations/he/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -2889,7 +2889,7 @@ msgstr ""
 
 #: src/cryptoadvance/specter/templates/settings/hwi_settings.jinja:89
 msgid "Add the domain"
-msgstr ""
+msgstr "הוסף לתחום"
```
Comment (ben-kaufman) [inline src/cryptoadvance/specter/translations/he/LC_MESSAGES/messages.po:2892]: I think הוסף את הכתובת would be better in the context of this

File: src/cryptoadvance/specter/translations/he/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -3002,15 +3002,15 @@ msgstr ""
 
 #: src/cryptoadvance/specter/templates/settings/tor_settings.jinja:95
 msgid "Allows access to Specter from anywhere."
-msgstr ""
+msgstr "אפשר גישה לספקטר מכל מקום."
```
Comment (ben-kaufman) [inline src/cryptoadvance/specter/translations/he/LC_MESSAGES/messages.po:3005]: אפשר -> מאפשר

File: src/cryptoadvance/specter/translations/he/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -3782,15 +3782,15 @@ msgstr ""
 
 #: src/cryptoadvance/specter/templates/wallet/send/new/wallet_send.jinja:63
 msgid "still in unsigned transactions."
-msgstr ""
+msgstr "טרנזקציה שאינה חתומה עדיין."
 
 #: src/cryptoadvance/specter/templates/wallet/send/new/wallet_send.jinja:71
 msgid "Unit:"
-msgstr ""
+msgstr "יחידה מידע:"
```
Comment (ben-kaufman) [inline src/cryptoadvance/specter/translations/he/LC_MESSAGES/messages.po:3789]: typo should be : יחידת מידה

File: src/cryptoadvance/specter/translations/he/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -3800,92 +3800,92 @@ msgstr ""
...
 
 #: src/cryptoadvance/specter/templates/wallet/send/new/wallet_send.jinja:83
 msgid "Remove recipient"
-msgstr ""
+msgstr "הסר נמען"
 
 #: src/cryptoadvance/specter/templates/wallet/send/new/wallet_send.jinja:84
 msgid "Calculate estimated fee"
-msgstr ""
+msgstr "חשב הערכת עמלה"
```
Comment (ben-kaufman) [inline src/cryptoadvance/specter/translations/he/LC_MESSAGES/messages.po:3817]: חשב עמלה משוערת

Might be better here

**Review by ben-kaufman — CHANGES_REQUESTED** [review]
Looks good! I added a few comments on typos and suggestions for improvement

File: src/cryptoadvance/specter/translations/he/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -2703,36 +2703,36 @@ msgstr ""
...
+msgstr "ברירת מחדל"
 
 #: src/cryptoadvance/specter/templates/settings/auth_settings.jinja:29
 msgid "Registration Link Timeout (hours)"
 msgstr ""
 
 #: src/cryptoadvance/specter/templates/settings/auth_settings.jinja:30
 msgid "Default username is 'admin', password 'admin'."
-msgstr ""
+msgstr "ברירת הממחדל לשם משתמש הוא 'admin', ולסיסממה 'admin'"
```
[the remark this answers is not in the document]
Comment (Sh0ham) [reply src/cryptoadvance/specter/translations/he/LC_MESSAGES/messages.po:2719]: Fucking MacBook butterfly keyboard :(

File: src/cryptoadvance/specter/translations/he/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -1931,11 +1931,11 @@ msgstr ""
 #: src/cryptoadvance/specter/templates/includes/tx-table.html:161
 #: src/cryptoadvance/specter/templates/wallet/send/pending/components/pending_psbt_table.jinja:15
 msgid "Time"
-msgstr ""
+msgstr "זמןמ"
```
[the remark this answers is not in the document]
Comment (ben-kaufman) [reply src/cryptoadvance/specter/translations/he/LC_MESSAGES/messages.po:1934]: Still not resolved?

**Comment by ben-kaufman** [comment]
Comments still don't seem to be resolved... Also need to run:
pybabel update -N --no-wrap -i babel/messages.pot -d src/cryptoadvance/specter/translations -l he

**Pull request closed without merging**